site/src/pages/components/accordion.explainer.mdx

```diff
@@ -151,6 +154,21 @@ violate the exclusivity semantics?
 (This applies both to insertion by the parser
 and insertion through other methods like setting `innerHTML`.)
 
+**Proposed solution**: Enforce exclusivity only for
+user interactions,
```

Comment:
I'm sure there will be discussion of this behavior, but I'm supportive. Sounds reasonable.